### DISCUSSION TITLE
lotus 节点 可以启动多个吗？一个对内一个对外

### DISCUSSION THREAD

**Discussion by wxd1343401** [filecoin-project/community-china#92] [新手上路]
lotus  daemon 可以启动多个吗？一个对内一个对外

**Comment by beck-8** [comment]
可以

**Reply by wxd1343401** [reply]
怎么配置呢，能给我一些建议吗？

**Comment by maxvint** [comment]
@wxd1343401  可以参考这一篇：https://github.com/filguard/lotus-ops/blob/master/documents/distributed-miner-configuration.md#72-%E5%88%86%E5%B8%83%E5%BC%8Fminer%E5%92%8Cdaemon%E4%B9%8B%E9%97%B4%E5%A6%82%E4%BD%95%E8%BF%9E%E6%8E%A5

**Comment by dcshy** [comment]
自有技术团队近百人， 机房密封日产能2p+ ， 独立节点集群方案， 可加vx：IPFS-Shy

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308,另接代封装业务，矿池搭建和运维服务，价格美丽，提供教学指导
